Dockerfile.allinone

```diff
@@ -0,0 +1,10 @@
+ARG CADENCE_IMAGE=ubercadence/server:latest
```

Comment:
Is it also standard to have Dockerfile.allinone and Dockerfile.CLI in the root? Can we only have one? Otherwise the root looks messy.

Reply:
Multi stage Dockerfiles can be built until a specific target, so we can merge the three into one, but it would make the build process slightly more complicated. I don't think there is any rule for multiple dockerfiles, but I don't find it messy at all.

That being said, the production and the allinone images could also be merged and configured with build arguments (`--build-arg ALLINONE=true`). Would it be okay to have one for the cli and one for the two server images?

Reply:
I think using that argument would be a better idea. CLI doesn't have to use different dockerfile

Reply:
> CLI doesn't have to use different dockerfile

What do you mean? I proposed two separate solutions, the build arg one would still have a separate dockerfile for the cli. Alternatively, we can get rid of the cli dockerfile altogether, but that would require some documentation changes (prepend cadence commands with `cadence`)

Reply:
I think this one is resolved by the single dockerfile